lib/DAV/Auth/Backend/Esn.php

```diff
@@ -91,7 +91,8 @@ private function decodeResponseV2($response) {
             return [false, null];
         }
 
-        $user = json_decode($response->getBodyAsString())[0];
+        # ISSUE-31: /api/users?email=username@linagora.com returns only 1 user instead of a list of user
+        $user = json_decode($response->getBodyAsString());
```

Comment:
Please no behaviour change.

I believe the code is right. And that the test is outdated.